File: plugins/OrchidSwagger/src/main/java/com/eden/orchid/languages/SwaggerResourceSource.java
package com.eden.orchid.languages;

import com.eden.orchid.api.OrchidContext;
import com.eden.orchid.api.resources.resourceSource.PluginResourceSource;

import javax.inject.Inject;

public class SwaggerResourceSource extends PluginResourceSource {

    @Inject
    public SwaggerResourceSource(OrchidContext context) {
        super(context, 100);
    }

}

File: plugins/OrchidSwagger/src/main/kotlin/com/eden/orchid/swagger/SwaggerResourceSource.kt
package com.eden.orchid.swagger

import com.eden.orchid.api.OrchidContext
import com.eden.orchid.api.resources.resourceSource.PluginResourceSource

import javax.inject.Inject

class SwaggerResourceSource @Inject
constructor(context: OrchidContext) : PluginResourceSource(context, 100)

